dashboard/test/lib/services/script_seed_test.rb

```diff
@@ -1121,6 +1121,7 @@ def create_script_tree(
           create :lesson, lesson_group: lg, script: script, name: name, key: name, overview: "overview #{m + 1} #{n + 1}", has_lesson_plan: true
         end
       end
+      script.reload
 
       (1..num_resources_per_script).each do |r|
         resource = create :resource, key: "#{script.name}-resource-#{r}", course_version: course_version
```

Comment:
just noticing... should the blocks that create resources here be moved below `next unless course_version`?

Reply:
oh totally! I'll add that in this PR as it shouldn't be a functional change

Reply:
hm actually, that `next unless course_version` is in the loop to create lessons, which isn't where these belong. I'll just add a specific check around these blocks

Reply:
ah, good catch. updates look good!